sklearn/tests/test_common.py

```diff
@@ -379,3 +381,37 @@ def test_transformer_n_iter():
 
         if hasattr(estimator, "max_iter") and name not in external_solver:
             yield check_transformer_n_iter, name, estimator
+
+
+# addresses #4467, #4465 and #4461
```

Comment:
Don't put such a comment in the source code. It will be meaningless in a few years.